contentcuration/contentcuration/frontend/channelEdit/components/ResourcePanel.vue

```diff
@@ -35,9 +35,9 @@
         </VFlex>
         <VSpacer />
         <!-- Slot for elements like edit button -->
-        <VFlex grow>
+        <div style="min-width: max-content;">
```

Comment:
Is there another way to achieve this effect? `max-content` was introduced in the last ten major versions of Edge, and we don't do comprehensive version checking.